client/my-sites/stats/activity-log-banner/error-banner.jsx

```diff
@@ -13,9 +13,17 @@ import { isUndefined } from 'lodash';
  */
 import ActivityLogBanner from './index';
 import Button from 'components/button';
+import HappychatButton from 'components/happychat/button';
+import Gridicon from 'gridicons';
 import TrackComponentView from 'lib/analytics/track-component-view';
+import { recordTracksEvent } from 'state/analytics/actions';
 import { dismissRewindRestoreProgress as dismissRewindRestoreProgressAction } from 'state/activity-log/actions';
 
+const happyChatBackupTracking = () =>
+	recordTracksEvent( 'calypso_activitylog_error_banner_backup' );
+const happyChatRestoreTracking = () =>
+	recordTracksEvent( 'calypso_activitylog_error_banner_restore' );
```

Comment:
it's a bit surprising to see these up here, what's the preference for naming these at the top vs. inlining them in `connect()` as is common elsewhere in Calypso?

Reply:
Updated to follow common pattern.